ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuite.scala

```diff
@@ -126,4 +126,41 @@ private[testtool] abstract class LedgerTestSuite(val session: LedgerSession) {
     }
   }
 
+  implicit def diffShowSeq[T](implicit diffShowT: DiffShow[T]): DiffShow[Seq[T]] =
+    new DiffShow[Seq[T]] {
+      override def show(t: Seq[T]): String = t.toString
+
+      override def diff(left: Seq[T], right: Seq[T]): Comparison = {
+        val leftList = left.toList
```

Comment:
Minor: why the conversion? If you need to convert, probably getting the iterator is a better choice, performance-wise (one scan over the collection instead of five).

Reply:
I'm converting because the `constructorOption` function requires a `List`. I do not know why, exactly; a `Seq` should be more than enough.

You're right about the performance. I thought, for some reason, that Scala's `List` was lazy, but apparently I need to use `Stream` or `Iterator` (as you say) to get that benefit. Will fix.